src/Handlers/BookingsStateStatusTransitionsHandler.php

```diff
@@ -137,10 +154,32 @@ protected function _prepareStatusTransitions($statusTransitions)
     protected function _prepareTransitionsLabels($transitionsLabels)
     {
         $translatedLabels = [];
-        foreach ($transitionsLabels as $transitionKey => $transitionLabel) {
-            $translatedLabels[$transitionKey] = $this->__($transitionLabel);
+        foreach ($transitionsLabels as $statusKey => $transitionsLabelsMap) {
```

Comment:
Looks like the docs of this method need to change, because I couldn't really understand what it does or returns from the descriptions.